dataactvalidator/config/sqlrules/a32_appropriations.sql

```diff
@@ -9,10 +9,9 @@ SELECT approp.row_number,
 FROM appropriation AS approp
 WHERE approp.submission_id = {0}
 	AND EXISTS (
-		SELECT other.row_number,
-			other.tas
+		SELECT 1
 		FROM appropriation AS other
 		WHERE other.row_number <> approp.row_number
-			AND other.tas IS NOT DISTINCT FROM approp.tas
+			AND other.tas_id IS NOT DISTINCT FROM approp.tas_id
```

Comment:
Can do and understood about the noise. There's the potential that we wouldn't show an A32 error if A1 was also in error, but that seems better than showing an A32 error _whenever_ A1 errors.